columnLayout.js

```diff
@@ -43,10 +41,16 @@ var countBlock = function(){
 		start = middleWidth - start;
 	}
 
+	// This default value sets top space from where this block starts $('.navbar') is the selecter to select top element
+	var topEle = parseInt($('.navbar').css('height'), 10) + parseInt($('.wrapper').css('margin-top'), 10);
```

Comment:
In my test code i have a header jumbotron and other elements that change size according to screen size. So some times ".qsb" blocks get on top of the div that is above them.
So rather than giving initial fix height of 60px that you had used i gave it variable height calculating size of the div above (".navbar") and top margin of ".wrapper" div. So our dynamic blocks never ever overlap the navbar even when navbar changes height.